export.js

```diff
@@ -135,6 +135,9 @@ Avatar = {
   // Get the url of the user's avatar
   getUrl: function (user) {
 
+    // Default to the currently logged in user, unless otherwise specified.
+    if (!user) user = Meteor.users.findOne(Meteor.userId())
```

Comment:
I still think you need this:

```
if (!user) user = Meteor.user()
```

Otherwise it won't update if a custom avatar URL changes.